src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/DeleteToolbarAction.js

```diff
@@ -27,12 +27,13 @@ export default class DeleteToolbarAction extends AbstractFormToolbarAction {
         form: Form,
         router: Router,
         locales: ?Array<string>,
-        options: {[key: string]: mixed},
+        options: { [key: string]: mixed },
```

Comment:
That's not how we usually write that... Is there maybe a ESLint rule missing resp. not working anymore? :thinking: